frontend/src/components/FormFooter.jsx

```diff
@@ -36,9 +36,9 @@ export default function FormFooter({
         <button
           className="button-post"
           onClick={handleSubmit}
-          disabled={length === null || length === 0}
+          disabled={length === undefined || length === 0}
         >
-          {buttonText}
+          {buttonText.includes("comment") ? "comment" : "post"}
```

Comment:
Please could these be capitalised?

`{buttonText.includes("comment") ? "Comment" : "Post"}`